src/containers/App/App.js

```diff
@@ -1,29 +1,41 @@
 import React from 'react';
 import { func } from 'prop-types';
 import { translate } from 'react-i18next';
+import { Router } from '@reach/router';
+import { Home } from '../../pages';
```

Comment:
No need to use `../../`, just import the home page like this:

```
import Home 'pages/home/Home';
```

Reply:
how is this enabled?

Reply:
That's a good question, we enabled it by adding this configuration to webpack: https://github.com/Coding-Coach/coding-coach/blob/development/scripts/config-overrides.dev.js#L30